### PR TITLE
Sync with Dolby

### PR DESCRIPTION
**Pull request by mstratford**

### CODE REVIEW

**Comment by michael-grace** [comment]
gonna see if it's even mergable

**Comment by michael-grace** [comment]
oh it can be. but it should be tested first given the mess it's gone through

File: scripts/stateserver.py [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+# normal shows in real studios aren't currently a thing!!!
+import subprocess
+from flask import Flask, jsonify, request
+from flask_cors import CORS
+import requests
+import datetime
+import random
+from telnetlib import Telnet
+
+api_key = "y2u%H%hpZID603j41Clkfks23k2dkglfjkfe3k4k2k4lk23qj2"
```
Comment (michael-grace) [inline scripts/stateserver.py:13]: don't worry, i've revoked this now 😂 😂 😂

**Comment by mstratford** [comment]
Please can we merge this <3

**Pull request merged**